lib/policies/cors/index.js

```diff
@@ -5,8 +5,8 @@ module.exports = {
     type: 'object',
     properties: {
       origin: {
-        type: ['string', 'boolean', 'array'],
-        items: { type: 'string' },
+        type: ['string', 'boolean', 'array', 'object'],
+        items: { type: ['string', 'object'] },
```

Comment:
This will do, but it is not exactly the solution I would like to employ. I think we should add yet another layer of validation on that to make sure the provided object is a `RegExp` object. Maybe we could use [instanceOf](https://github.com/epoberezkin/ajv-keywords#instanceof) plugin to make it happen?

Reply:
Sorry fat fingered resolve button. I understand, I have read js-yaml that we need to validate should we pass types other than the primitive ones.